### ISSUE TITLE
Errors are hard to read

### ISSUE THREAD

**Issue by arturogh** [framer/syntax-vsc#5]
Seems like the text color for validation errors is the default color. against the color used for errors it is hard to read the message.

![microsoftteams-image 2](https://user-images.githubusercontent.com/7030831/44809377-5fbbaa80-ab83-11e8-9cf8-5dba035157fa.png)

**Comment by DannyJJK** [comment]
I don't think this theme is updated anymore. It's missing various things, such as the built-in Git edit/add/deletion colors where the line numbers are, plus the highlight color inside the search file box is really hard to see.

**Comment by arturogh** [comment]
There's a fix for this in a PR now 😄 #6
